src/write_eval_config.R

```diff
@@ -116,14 +116,27 @@ write_eval_config <- function(locations, forecast_dates,
     dates_to_exclude = c("2024-01-30", "2024-01-31", "2024-02-01", "2024-02-02")
   )
```

Comment:
Delete this?

Reply:
These are for the hospital admissions but yeah I don't think we should include these since we arent doing this everywhere, will delete.